main/linux.py

```diff
@@ -18,11 +17,14 @@ def install_libre():
     ec.INSTALLED_LIBRE = True
 
 
-def convert_linux(filepath: str, files, converted_file_path: str, has_type_in_request):
+def convert_linux(
+    filepath: str, files, converted_file_path: str, has_type_in_request: dict
```

Comment:
type of files argument is not defined

Reply:
Also, type of has_type_in_request should be replaced with Dict[..., ...] **if possible**, where `...` is types of key and value respectively